include/triton/Dialect/TritonGPU/IR/TritonGPUAttrDefs.td

```diff
@@ -1228,16 +1231,15 @@ a's opIdx is 0, b's opIdx is 1.
 
 The parend field in DotOperandEncodingAttr is the layout of d.
 
-For MMA v1, an additional attribute `isMMAv1Row` determines whether e.g. the a operand is used
-in the context of an mma.884.row.col or an mma.884.col.col operation. See the PTX ISA documentation
-section 9.7.13.4.1 for more details.
+kWidth defines number of consecutive elements stored by one thread along k dimension.
+Some layouts do not use this parameter, because they have fixes number or use all elements of tensor along K dim.
```

Comment:
```suggestion
Some layouts do not use this parameter, either because they have a fixed number of elements along the K dim, or they use all elements of the tensor along the K dim.
```